<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 74, 135);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E55735;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#F93A0C;
color:black
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E9E286;
color:white;
font-size:20px;
}
QPushButton:pressed
{
background-color:#52C11D;
color:black
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equalsto">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(114, 159, 207);
qproperty-alignment:&quot;AlignRight&quot;;
}
</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_sign">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equalsto_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#BEC3B5;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#80827C;
color:black
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid black;
background-color:#E55735;
color:black;
font-size:20px;
}
QPushButton:pressed
{
background-color:#F93A0C;
color:black
}</string>
    </property>
    <property name="text">
     <string>Go To Converter</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
